electives/18729.md

```diff
@@ -0,0 +1,43 @@
+# 18-429/729: Board-Level RF Systems for IoT
+
+| Category | Difficulty |
+|:-:       | :-:        |
+| Homeworks  | 6     |
+| Labs  | 4     |
+| Quizzes  | 6     |
+
+This course explores RF (radio-frequency) systems design from several perspectives. It discusses transmission lines, antennas, transceiver design, modulation techniques, information theory, SDRs (software-defined radios), phased arrays, MIMO (multiple-input multiple-output), and more. It's a great way to get an overview of various topics related to wireless systems design.
+
+If you're more interested in circuit-level (i.e. transistors) design for wireless systems, 18-723 is a good course to look into. If you're interested in more of the software and theory behind wireless systems, then courses like 18-741 or 18-750 can be good instead. This course is somewhere in-between, not quite going into IC-level design nor looking at the network layer, but instead exploring systems-level design. Check out the Prerequisites section for more information on what you need going into 18-729.
+
+The course has lectures twice per week and a lab or recitation once per week. Labs and recitations usually alternate weeks, with recitations intended to help review for quizzes. Be aware that Fall 2022 was the first time that the course was offered; thus, a lot of the course structure (labs, lectures, etc.) may change in the next iteration. The course itself may also be renumbered from 18-429/729 to something else in Fall 2023.
+
+## Prerequisites
+
+Nominally, the course requires 18-290 and 18-320 as prerequisites. Signal processing is indeed a key component in RF design, so having a strong background (and perhaps having taken 18-491 as well) can prove very helpful. Regarding 18-320, the content in that course might be overkill for understanding 18-729. 18-729 does not go into transistor-level design, since 18-723 explores this. Thus, you only need the fundamentals from 18-220 to understand 18-729 -- RLC circuits and basic electromagnetics, for example. 18-729 explores transmission lines and antenna design from first principles, so you don't need to be an expert in circuits to do well in the course. The prerequisite of 18-320 may change to 18-220 in the next iteration of the course.

```

Comment:
If you can, I would link the documents for these other classes that you mention here.